### PR TITLE
Add Fixed Fee setting for cash-out transactions

### PR DESCRIPTION
**Pull request by siiky**

### CODE REVIEW

File: ui/src/app.js [hunk 1]
```diff
@@ -111,7 +111,10 @@ function processData (data) {
   if (data.sent && data.total) setPartialSend(data.sent, data.total)
   if (data.readingBill) readingBill(data.readingBill)
   if (data.cryptoCode) translateCoin(data.cryptoCode)
-  if (data.tx && data.tx.cashInFee) setFixedFee(data.tx.cashInFee)
+  if (data.tx) {
+    if (data.tx.cashInFee) setFixedFee(data.tx.cashInFee)
+    else if (data.tx.cashOutFee) setFixedFee(data.tx.cashOutFee)
+  }
```
Comment (RafaelTaranto) [inline ui/src/app.js:118]: This needs to get built, `npm run build`

File: lib/brain.js [hunk 2]
```diff
@@ -3155,16 +3155,7 @@ Brain.prototype.updateBillScreen = function updateBillScreen (blockedCustomer) {
...
-  var billUpdate
-  // BACKWARDS_COMPATIBILITY 7.5.0-beta.1
-  const serverVersion = this.trader.serverVersion
-  if (!serverVersion || semver.lt(serverVersion, '7.5.0-beta.1')) {
-    billUpdate = Tx.billUpdateDeprecated(bill)
-  } else {
-    billUpdate = Tx.billUpdate(bill)
-  }
-
-  return this.fastUpdateTx(billUpdate)
```
Comment (RafaelTaranto) [inline lib/brain.js:3174]: Yeah, it's time to remove this. But we need to thing of something for this scenario in specific. Maybe throw a error, maybe do it generically on the machine if versions are too far apart. 

@CrypticaScriptura What do you think?

File: lib/tx.js [hunk 3]
```diff
@@ -23,20 +23,22 @@ function mergeTx (oldTx, updateTx) {
...
       _.reduce((acc, v) => acc.add(toCrypto(mergedTx, BN(v.fiat).minus(v.cashInFee))), BN(0), bills),
       cryptoCode),
-    cashInFeeCrypto: truncateCrypto(toCrypto(mergedTx, cashInFee), cryptoCode)
+    cashInFeeCrypto: truncateCrypto(toCrypto(mergedTx, fee), cryptoCode)
   })
 
   const cashOutNewFields = () => ({
     fiat: oldTx.fiat.add(updateTx.fiat || 0),
-    cryptoAtoms: truncateCrypto(toCrypto(mergedTx, oldTx.fiat.add(updateTx.fiat || 0)), cryptoCode)
+    cryptoAtoms: truncateCrypto(toCrypto(mergedTx, oldTx.fiat.add(updateTx.fiat || 0).minus(fee)), cryptoCode),
```
Comment (RafaelTaranto) [inline lib/tx.js]: Shouldn't we be adding the fee?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/tx.js (hunk 3 above)
Comment (siiky) [reply lib/tx.js]: Yeah, nice catch!

File: lib/brain.js (hunk 2 above)
Comment (CrypticaScriptura) [reply lib/brain.js:3174]: Likely time for a version discrepancy screen on the machine that prevents operation entirely if the particular machine's version is sufficiently distant from that of the server. Would be preferable if reserved for major version releases (as this is, v9.0.0).

**Pull request merged**